dom/buyer.js

```diff
@@ -0,0 +1,57 @@
+const pureFuncs = require("../utlis/function");
+let products = JSON.parse(localStorage.getItem("productArray"));
+let cart = [];
```

Comment:
A good practice to follow is to declare all your variables with `const` key word unless you intend to re-assign it somewhere else in the code then use the `let` key word.

**Note:** even if you assign an array to a variable declared with `const` key word you can mutate it by pushing elements to it for example. Mutability and reassignment are different concepts.

check this out: [Reassignment vs Mutability](https://dev.to/boywithsilverwings/reassignment-vs-mutability-21ob)